crates/ide/src/runnables.rs

```diff
@@ -106,6 +110,92 @@ pub(crate) fn runnables(db: &RootDatabase, file_id: FileId) -> Vec<Runnable> {
     res
 }
 
+pub(crate) fn related_tests(
```

Comment:
This needs a `// Feature: Related Tests` comment, otherwise no one will know that the thing exists )

Reply:
Yeah, right!